src/internal/snapshot/pretty/pretty.go

```diff
@@ -22,7 +22,7 @@ func PrintDetailedSnapshotInfo(info *snapshot.SnapshotInfo) error {
 	t, err := template.New("SnapshotInfo").Funcs(funcMap).Parse(
 		`ID: {{.Id}}
 Chunkset: {{.ChunksetId}}
-Created: {{prettyAgo .Created}}{{if .PachydermVersion}}
+Created: {{prettyAgo .CreatedAt}}{{if .PachydermVersion}}
```

Comment:
Actually, it would be good to show the SQL Dump Fileset here.

Reply:
oh makes sense. I'll fix this.